Sources/Utilities/Storage/Session/PersistentSessionManager.swift

```diff
@@ -139,6 +148,18 @@ final class PersistentSessionManager: SessionManager {
     }
 
     func resumeSession() throws {
+        guard expiryDate != nil else {
+            throw PersistentSessionError.noSessionExists
+        }
+        
+        guard isSessionValid else {
+            throw TokenError.expired
+        }
+        
+        guard hasNotRemovedLocalAuth else {
+            throw PersistentSessionError.userRemovedLocalAuth
```

Comment:
Please can we add a test that throws here? (and the `TokenError` above).
Would be good to get the coverage showing as high as possible 🙂